db/migrate/20150629192749_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.string :body, null: false
+      t.integer :rating, null: false
+      t.integer :spot_id
+      t.integer :user_id
```

Comment:
Does it make sense to have a record in the reviews table that doesn't have an associated user? What about an associated spot?

We probably want `null: false` constraints on the database, here.